src/main/java/lottoauto/model/Lotto.java

```diff
@@ -26,25 +30,15 @@ public List<Integer> getNumbers() {
         return this.numbers;
     }
 
-    @Override
-    public boolean equals(Object o) {
-        if (this == o) {
-            return true;
+    private void lottoValidation(List<Integer> lotto) {
+        if (lotto.size() != 6) {
```

Comment:
로또와 헷갈릴 수 있으니 변수명을 적절하게 변경하면 좋을 것 같습니다.

```suggestion
    private void lottoValidation(List<Integer> lottoNumbers) {
        if (lotto.size() != 6) {
```